bin/drush-install.sh

```diff
@@ -36,32 +36,6 @@ org.civicrm.shoreditch,\
 org.civicrm.bootstrapcivihr,\
```

Comment:
There's a problem with this git commit message. The title says "Move Job enabling to Extensions", but then you look at the description and it's clear that you're doing more than that with this comment. You also made change to the upgraders, which have nothing to do with the scheduled jobs. In the future, for cases like this, please put the changes in separate commits.